pennylane/ops/math/exp.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-few-public-methods,function-redefined
+
+import pennylane as qml
+
+
+class Exp(qml.operation.Operator):
```

Comment:
For now I implemented the classes straight. I'd rather find what's common, and only make a base class if this helps. I find that base classes can obscure many things...

Reply:
I'll just caution that they won't be processed from the queue into the tape operations if they inherit from `Operator` instead of `Operation`.  

Maybe we need to update [this line](https://github.com/PennyLaneAI/pennylane/blob/d201dd52def0dfa44efd485e06ea06defda22dc0/pennylane/tape/tape.py#L416) in `QuantumTape._process_queue` instead.